### DISCUSSION TITLE
Using BSON ObjectId and Cuid in the same field

### DISCUSSION THREAD

**Discussion by NoxTav** [prisma/prisma#15254] [Q&A] [answered]
Hi,

We are thinking of adding Prisma to our stack. We currently have a MongoDB database and all our ids are BSON ObjectId.

As we will copy the data to an SQLite database (as part of an offline mode), we need to have a type that will work with both.

Can having BSON ObjectId in a Mongo database where we say the ids are Cuids lead to issues?

Note: we will be using Prisma in Rust.

**Comment by austincrim** [comment, marked as answer]
Hey @NoemieTav 👋,

I think it should be possible to specify the `_id` field as a CUID in your schema. Then it should work across Mongo and SQLite. ObjectId's will not work outside of Mongo, as far as I know.

```prisma
model User {
  id  String @map("_id") @default(cuid())
}
```

I'm not aware of any potential issues with this approach, but if you run into anything, let us know 👍.
